src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioDiagnosticListTableCommandHandler.cs

```diff
@@ -103,5 +149,114 @@ private void OnRemoveSuppressions(object sender, EventArgs e)
         {
             _suppressionFixService.RemoveSuppressions(selectedErrorListEntriesOnly: true, projectHierarchyOpt: null);
         }
+
+        private void OnErrorListSetSeveritySubMenuStatus(object sender, EventArgs e)
+        {
+            // For now, we only enable the Set severity menu when a single configurable diagnostic is selected in the error list
+            // and we can update/create an editorconfig file for the configuration entry.
+            // In future, we can enable support for configuring in presence of multi-selection. 
+            var command = (MenuCommand)sender;
+            var selectedEntry = TryGetSingleSelectedEntry();
+            command.Visible = selectedEntry != null &&
+                !SuppressionHelpers.IsNotConfigurableDiagnostic(selectedEntry) &&
+                TryGetPathToAnalyzerConfigDoc(selectedEntry, out _) != null;
+            command.Enabled = command.Visible && !KnownUIContexts.SolutionBuildingContext.IsActive;
+        }
+
+        private void SetSeverityHandler(object sender, EventArgs args)
```

Comment:
Can we add tests for this method?

Reply:
> Would integration tests be useful/possible?

Yes, but we don't have necessary test infrastructure to test commands executed from analyzers node or error list diagnostic context menu. That would be a much bigger test item.

> Can we add tests for this method?

This would again need a more bigger test change. Note that the core functionality that adds/updates editorconfig here, i.e. `ConfigurationUpdater.ConfigureSeverityAsync` invoked below, is already tested via the unit tests for our light bulb code actions to configure severity.

Reply:
Ah, I thought that might be the case. Thanks!